hs_collection_resource/views.py

```diff
@@ -0,0 +1,106 @@
+import json
+import logging
+
+from django.http import HttpResponseRedirect, HttpResponseNotFound, HttpResponse, JsonResponse
+
+from hs_core import hydroshare
+from hs_core.views import _set_resource_sharing_status
```

Comment:
@pkdash I think the client.post() only works in testing code, is it possible to create a new Client object in an existing controller function and call client.post()?